mobile/voice_journal_app/lib/main.dart

```diff
@@ -1,17 +1,20 @@
 import 'package:flutter/material.dart';
 import 'package:voice_journal_app/theme.dart';
 import 'home.dart';
-import 'theme.dart';
-
-void main() {
+import 'package:hive_flutter/hive_flutter.dart';
+import 'schema.dart';
+ 
+void main()async { 
+  await Hive.initFlutter(); //Initialize hive for flutter crucial step
+  Hive.registerAdapter(RecordingAdapter()); //Register the adapter, essentially telling Hive how to read and write our Recording information into a box
+  await Hive.openBox<Recording>('recordings');
```

Comment:
Should be run in the constructor for the Recording page, otherwise Hive has the `RecordingAdapter` registered for the entire application and if another page changes it the Recording page calls to Hive will fail.

Reply:
Let's remove this in favor of putting it in the Future function in the home page